src/sage/interfaces/r.py

```diff
@@ -1274,7 +1268,7 @@ def plot(self, *args, **kwds):
         persistent in your own code::
 
             sage: from tempfile import TemporaryDirectory
-            sage: with TemporaryDirectory() as d: # optional - rpy2, rgraphics
+            sage: with TemporaryDirectory() as d:               # optional - rgraphics
             ....:     _ = r.setwd(d)
```

Comment:
Where is `rgraphics` feature defined?

Reply:
It isn't anywhere

Reply:
Then does `# optional - rgraphics` work?

Reply:
Well, people who know about it can use `sage -t --optional=sage,rpy2,rgraphics`, I suppose.

Reply:
Perhaps I forgot how the `optional` mechanism is supposed to work...

Reply:
OK. What you said seems the best we can suppose.